scripts/linters/python_linter.py

```diff
@@ -60,15 +60,15 @@ def write(self, message):
         """Writes the given message to messages list.
 
         Args:
-            message: str. Message returned by the pylinter.
+            message: str. Message returned by the method.
```

Comment:
This is confusing, because "the method" seems to refer to write(). Please be clearer and write from the perspective of this class. Perhaps "The message to be written."

Reply:
Done.